modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerPlugin.java

```diff
@@ -153,6 +156,16 @@ public void execute(Configuration configuration) {
 	protected void addDependenciesPoshiRunner(
 		Project project, PoshiRunnerExtension poshiRunnerExtension) {
 
+		File poshiRunnerPropertiesFile =
+			poshiRunnerExtension.getPoshiRunnerProperties();
+
+		Properties poshiRunnerProperties = GUtil.loadProperties(
+			poshiRunnerPropertiesFile);
+
+		GradleUtil.addDependency(
```

Comment:
Here you're adding the properties file to the `poshiRunner` configuration, which is wrong: a [configuration](https://docs.gradle.org/current/userguide/artifact_dependencies_tutorial.html#configurations) in Gradle is a list of dependencies, not the "configuration" of the task... You should add the settings you find in the properties files next to the place where we apply the properties we find in `poshiRunner.poshiProperties` (look at `PoshiRunnerExtension`).